tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -153,6 +150,25 @@ public ImmutableTracerSettings(TracerSettings settings)
         /// <seealso cref="ConfigurationKeys.ServiceVersion"/>
         public string ServiceVersion { get; }
 
+        /// <summary>
+        /// Gets the application's git repository url.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.GitRepositoryUrl"/>
+        public string GitRepositoryUrl { get; }
```

Comment:
nit: I'm not sure these need to be public. Doing so exposes extra surface area that customers _could_ depend on, without any real benefit

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/commit/989c9c823ebb86da47f43eeaf48a2466a04e38af